engines/bladerunner/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
-# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine bladerunner "Blade Runner" yes "" "" "16bit highres"
+# add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
+add_engine bladerunner "Blade Runner" yes "" "" "16bit highres mt32emu"
```

Comment:
Blade Runner doesn't use MIDI beyond checking if a null device is selected, so MT32 emulation should't be needed.